yarn.lock

```diff
@@ -11230,11 +11226,6 @@ jquery.finger@*:
   resolved "https://registry.yarnpkg.com/jquery.finger/-/jquery.finger-0.1.6.tgz#24a481103c28ff73eebef3e39328403770eed0fa"
   integrity sha1-JKSBEDwo/3PuvvPjkyhAN3Du0Po=
 
-jquery.ui.widget@*:
-  version "1.10.3"
-  resolved "https://registry.yarnpkg.com/jquery.ui.widget/-/jquery.ui.widget-1.10.3.tgz#2ceecfc64995412643eeb9508d2197ea93c02fb3"
-  integrity sha1-LO7PxkmVQSZD7rlQjSGX6pPAL7M=
-
```

Comment:
omg. what's left using jquery at this point?

Reply:
Examples, tools, manage, a tiny bit left of settings.

Tools and settings will be easy, examples and manage annoying.